### DISCUSSION TITLE
How to delete this repository

### DISCUSSION THREAD

**Discussion by A14nDev** [skills/.github#209] [Q&A] [answered]
How to delete this repository, I can't find the delete button.
![image](https://user-images.githubusercontent.com/89789900/227719737-0566de1f-62df-49d5-88fa-d9eddabe1d0b.png)

**Comment by sinsukehlab** [comment, marked as answer]
It seems that you don't own this repository. You can't delete it.

**Reply by A14nDev** [reply]
OK, Thank you.

**Reply by A14nDev** [reply]
It's auto-created, and the owner is Global Classroom.

**Reply by sinsukehlab** [reply]
Thank you for selecting this as the answer.
You can generate your own repository from the template repository skills/introduction-to-github.

**Comment by cmwilson21** [comment]
@sinsukehlab Thanks for being so helpful in these forums! ✨ 

@huazihan Since it seems you've got an answer, I'm going to go ahead and close this discussion 💛
